workers/data_refinery_workers/processors/gene_convert.R

```diff
@@ -0,0 +1,92 @@
+###
+# gene_convert.R
+# Alex's Lemonad Stand Foundation
+# Childhood Cancer Data Lab
+#
+###
+
+# import magrittr pipe
+`%>%` <- dplyr::`%>%`
+
+#######################
+# The command interface!
+#######################
+
+library("optparse")
+library(data.table)
+library("dplyr")
+library("rlang")
+
+option_list = list(
+  make_option(c("-p", "--platform"), type="character", default="", 
+              help="Platform", metavar="character"),
+  make_option(c("-i", "--inputFile"), type="character", default="", 
+              help="inputFile", metavar="character"),
+  make_option(c("-o", "--outputFile"), type="character", default="", 
+              help="outputFile", metavar="character")
+); 
+
+opt_parser = OptionParser(option_list=option_list);
+opt = parse_args(opt_parser);
+
+platform <- opt$platform
+filePath <- opt$inputFile
+outFilePath <- opt$outputFile
+
+# Read the data file
+message("Reading data file...")
+suppressWarnings(data <- fread(filePath, 
+					stringsAsFactors=FALSE, 
+					sep="\t", header=TRUE, 
```

Comment:
Will these files have headers in our system? Sample tables from at least GEO do not generally have them. Processed data from ArrayExpress also tend to follow a bit of a different (multi-sample) format as far as I can tell.

Reply:
I _think_ the answer is yes based on `no_op.py` but I'm gonna leave this here so it can be confirmed.